### ISSUE TITLE
Page element fail color contrast ratio test

### ISSUE THREAD

**Issue by jeremyboggs** [scholarslab/scholarslab.org#519]
### Context
There are a number of elements that fail an accessibility test on color contrast ratio, between background and foreground colors. These appear to include any links that use the brighter teal color, and eyebrows that use a similar color or light gray. There are quite a few of these, so I won't add a full list here.

### Process
1. Run Accessibility audit in Chrome's Developer Tools on the home page and main section pages, and note specific elements that fail the test;
2. Find the appropriate CSS ruleset that govern's each element, and modify the colors in a way that retain uniformity across the site while still passing the contrast ratio test.

### Expected Result
Accessibility audits done on any page should pass for color contrast.

**Comment by amandavisconti** [comment]
Not applicable to /realignment color scheme
